docs/cert-manager.md

```diff
@@ -11,67 +39,187 @@ by [cert-manager](https://cert-manager.io/).
+  selfSigned: {}
+```
+
+```shell
+kubectl apply -f selfsigned-root-clusterissuer.yaml
+```
 
-Use the example to deploy a `MinIO` tenant using tls certificates generated by `cert-manager`, go into base folder of
-the operator project and run the following command.
+### Create tenant-1 namespace CA intermediary Issuer and
+In this step we will create the Intermediary Certificate Authority (CA) certificate used to issue certificates for MinIO.
 
-```bash
-kustomize build examples/kustomization/tenant-certmanager | kubectl apply -f -
+First let's create the `Issuer`:
+
+```yaml
+# tenant-1-ca-issuer.yaml
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: tenant-1-ca-issuer
+  namespace: tenant-1
+spec:
+  ca:
+    secretName: tenant-1-ca-secret
+```
+
+```shell
+kubectl apply -f tenant-1-ca-issuer.yaml
+```
+
+Now we are going to request a Certificate with the `isCA: true` spec, this will be our intermediary CA.
+
+```yaml
+# tenant-1-ca-certificate.yaml
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: tenant-1-ca-certificate
+  namespace: tenant-1
+spec:
+  isCA: true
+  commonName: tenant-1-ca
+  secretName: tenant-1-ca-secret
+  duration: 70128h # 8y
+  privateKey:
+    algorithm: ECDSA
+    size: 256
+  issuerRef:
+    name: selfsigned-root
+    kind: ClusterIssuer
+    group: cert-manager.io
+```
+
+```shell
+kubectl apply -f tenant-1-ca-certificate.yaml
+```
+
+## Deploy tenant 
+
+### Create certificate for Tenant
+
+We need to request `cert-manager` to issue a new certificate based on the following internal domains.
+
+```shell
+minio.<namespace>
+minio.<namespace>.svc
+minio.<namespace>.svc.<cluster domain>
+*.<tenant-name>-hl.<namespace>.svc.<cluster domain>
+*.<namespace>.svc.<cluster domain>
 ```
 
-This file request `cert-manager` to issue a new certificate based on the following internal domains.
+> [!IMPORTANT]
+> Replace `<tenant-name>`, `<namespace>` and `<cluster domain>` with the actual values for your MinIO tenant, Please note the 3 key element names:
+> * `tenant-name` is the name provided to your tenant this is the one provided in the `metadata.name` of the Tenant YAML, for this example it will be `myminio`.
+> * `namespace` is the namespace where the tenant is created, the `metadata.namespace` notes that in the Tenant YAML, for this example it will be `tenant-1`.
+> * `cluster domain` is the DNS assigned in your kubernetes cluster, typically it is `cluster.local`, check on your coredns configuration for the right one assigned to your kubernetes cluster.
+> For example querying `kubectl get configmap coredns -n kube-system -oyaml | yq ".data"`, this command could vary depending on the Kubernetes distribution like Openshift, Rancher, EKS, etc.
+
+Create a `Certificate` for the domains mentioned above:
 
 ```yaml
+# tenant-1-minio-certificate.yaml
 apiVersion: cert-manager.io/v1
 kind: Certificate
 metadata:
   name: tenant-certmanager-cert
-  namespace: minio-tenant
+  namespace: tenant1
```

Comment:
I guess there is a typo here that caused this to fail:
```suggestion
  namespace: tenant-1
```

Reply:
right, thanks